api/availability.toml

```diff
@@ -197,3 +197,20 @@ Get the stake table after `:height` leaves or after `:view_number`.
 
 Returns a map from staking public keys to amounts.
 """
+
+
+[route.get_block_summary]
+PATH = ["block/summary/:height"]
```

Comment:
I think it makes sense to index these summaries the same way we index the full blocks, ie we would also want to add `block/summary/hash/:hash` and `block/summary/payload-hash/:payload-hash`. But if this is not blocking the block explorer, we can save it for future work.

Reply:
Opting to save this for the future.